### PR TITLE
[feature] Add OWE encryption type

### LINKED ISSUE #193: [feature] OpenWrt: add support for OWE (opportunistic wireless encryption)
Depends on #192.

ieee80211w may be set to 2 (required) in order for owe to work, [I asked in the OpenWrt forum to double check](https://forum.openwrt.org/t/clarification-about-ieee80211w-and-wpa3-in-wireless-ap-configuration/103656).

The value to use in encryption is simply "owe", no other setting is required.

### PR DESCRIPTION
**Pull request by Aryamanz29**
Related to : https://github.com/openwisp/netjsonconfig/pull/251

Closes #193

### CODE REVIEW

**Comment by nemesisdesign** [comment]
Please reference the issue: https://github.com/openwisp/netjsonconfig/issues/193

File: netjsonconfig/schema.py [hunk 1]
```diff
@@ -416,6 +418,22 @@
...
         },
+        "encryption_owe": {
+            "title": "OWE",
+            "allOf": [
+                {"$ref": "#/definitions/encryption_mfp_property_required"},
+                {
+                    "properties": {
+                        "protocol": {
+                            "type": "string",
+                            "title": "encryption protocol",
```
Comment (nemesisdesign) [inline netjsonconfig/schema.py:429]:
```suggestion
                            "title": "Opportunistic Wireless Encryption",
```

File: netjsonconfig/schema.py [hunk 2]
```diff
@@ -416,6 +418,22 @@
...
+                            "enum": ["owe"],
+                            "options": {"enum_titles": ["OWE"]},
```
Comment (nemesisdesign) [inline netjsonconfig/schema.py:431]:
```suggestion
                            "options": {"enum_titles": ["Opportunistic Wireless Encryption"]},
```

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
This patch misses at least 1 render and 1 parse tests.

**Comment by kellerman** [comment]
If you want to do the transitioning to OWE/Open systems, then yes, these options also have to be supported:
owe_transition_ssid,option owe_transition_bssid,owe_transition_ifname
And maybe some more options, from a quick tell...
I personally don't use transitioning, It didn't work quite well for me.

**Review by nemesisdesign — APPROVED** [review]
I had to make a few adjustments in bfff240.

**Comment by nemesisdesign** [comment]
Thanks @kellerman @Aryamanz29 :+1: :clap: :fire:

**Pull request merged**